frameworks/moveos-stdlib/sources/account.move

```diff
@@ -16,8 +16,11 @@ module moveos_std::account {
       sequence_number: u64,
    }
 
-   /// ResourceAccount can only be stored under address, not in other structs.
-   struct ResourceAccount has key {}
+   // Account holder holds an account of the resource
+   struct AccountHolder has key {
```

Comment:
Do not need the Holder in framework. Framework provide function to create Object<Account>, the application can hold Object<Account> as resource account.